src/routes.tsx

```diff
@@ -83,8 +81,6 @@ export const getAppRoutes = (): AppRouteConfig[] => {
     { routes: collectRoutes },
     { routes: collectorProfileRoutes },
```

Comment:
I think if we also delete the consign routes from here - we would be at a great position

Reply:
Do you mean consignRoutes on line 83? It's actually /sell routes. I can rename it
Also, we still have /consign route that redirects to /sell -- I think we can already remove it. What do you think?

Reply:
I actually meant to delete the sell routes from the collector profile and have all of them all within the sellRoutes/consignRoutes. This way there would be only one route for submissions